sonic_installer/main.py

```diff
@@ -700,11 +700,11 @@ def cleanup():
 @click.option('--cleanup_image', is_flag=True, help="Clean up old docker image")
 @click.option('--skip_check', is_flag=True, help="Skip task check for docker upgrade")
 @click.option('--tag', type=str, help="Tag for the new docker image")
-@click.option('--warm', is_flag=True, help="Perform warm upgrade")
+@click.option('--advanced', is_flag=True, help="Perform advanced upgrade")
```

Comment:
We can't check this flag. Also, we cannot perform fast upgrade, only warm.

Reply:
Reverted, thanks.